attributes/default.rb

```diff
@@ -19,3 +19,25 @@
 #
 
 default['zabbix_ng']['zabbix_server'] = '127.0.0.1'
+default['zabbix_ng']['version'] = value_for_platform(
+  ['rhel','centos']   => {
+    "~> 7" => '3.0.0-1.el7',
+  },
+  'ubuntu'  => {
+    '~> 14' => '1:3.0.0-1+trusty'
```

Comment:
I do not think it's a good idea to version-lock repository packages. This will break as soon as there is a minor update or security fix.
For Ubuntu, the major version is specified using the repository URL, which is fine I think. Does this work differently on RHEL, or what was the reason you included this?